plugins/legacybanlist/q_shared.h

```diff
@@ -456,7 +456,7 @@ void MakeNormalVectors( const vec3_t forward, vec3_t right, vec3_t up );
 //int	PlaneTypeForNormal (vec3_t normal);
 
 void MatrixMultiply(float in1[3][3], float in2[3][3], float out[3][3]);
-void AngleVectors( const vec3_t angles, vec3_t forward, vec3_t right, vec3_t up);
+void Math_AnglesToVectors( const vec3_t angles, vec3_t forward, vec3_t right, vec3_t up);
```

Comment:
would be better to keep it on AngleVectors to be able to reference it in q3 sources